node/lib/package/PackageInfo.ts

```diff
@@ -79,6 +79,7 @@ export class PackageInfo {
                     description: translatorsJson[moduleName].description,
                     moduleName: moduleName,
                     onboarding: onboardingReference,
+                    onboardingFlow: [],
                     onboardingProperties: onboardingProperties,
```

Comment:
sure. Fixed populating onboardingInfo as well.

---
In reply to: [75431271](https://github.com/openT2T/opent2t/pull/22#discussion_r75431271) [](ancestors = 75431271)